sdks/python/apache_beam/options/pipeline_options_validator.py

```diff
@@ -250,6 +252,20 @@ def validate_sdk_container_image_options(self, view):
 
     return errors
 
+  def validate_worker_num(self, view):
+    """Validates that Dataflow worker number is valid."""
+    errors = self.validate_optional_argument_positive(view, 'num_workers')
```

Comment:
Should we validate that max_num_workers is positive (if set) as well?

Reply:
Done.